src/components/LessonInfoDisplay/LessonInfoDisplay.js

```diff
@@ -22,42 +49,49 @@ function LessonInfoDisplay(props) {
   }
 
   return (
-    <div className="LessonInfo">
-      <div className="InfoDisplay">
-        <div className="Column">
-          <div className="WordGroupImage"></div>
-          <div className="WordGroupName">{props.template}</div>
-        </div>
-        <div className="Column">
-          <div className="LessonDataTitle">WORDS</div>
-          <div className="LessonData">
-            {props.words.map((word, index) => (
-              <div key={index}>{word}</div>
-            ))}
-          </div>
-        </div>
-        <div className="Column">
-          <div className="LessonDataTitle">STUDENTS</div>
-          <div className="LessonData">
-            {props.studentNames.map((name, index) => (
-              <div key={index}>{name}</div>
-            ))}
+    <div className="lesson-info">
+      <div ref={ref}>
+        <Row>
+          <div className="info-display">
+            <div className="column">
+              <div className="word-group-display">
+                <div className="lesson-group-name">{props.template}</div>
+              </div>
+              <div className="word-display">
+                <div className="grey-box">
+                  <div className="word-group-name">
+                    {" "}
+                    {props.lesson.wordGroup}{" "}
```

Comment:
i'm pretty sure prettier did this but what would happen if you removed the `{" "}`s